src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -1,258 +1,655 @@
+    "object": "graphqldb.earth",
+    "type": null,
+    "parameters": null,
+    "key-fields": null
+  },
+  "graphql": {
+    "enabled": true,
+    "operation": null,
+    "type": {
+      "singular": "Earth",
+      "plural": "Earths"
+    }
+  },
+  "rest": {
+    "enabled": false,
+    "path": null,
+    "methods": []
+  },
+  "permissions": [
+    {
+      "role": "anonymous",
+      "actions": [
+        {
+          "action": "update",
+          "fields": {
+            "exclude": [
+              "*"
+            ],
+            "include": null
+          },
+          "policy": {
+            "request": null,
+            "database": null
+          }
+        },
+        {
+          "action": "read",
+          "fields": {
+            "exclude": [
+              "name"
+            ],
+            "include": [
+              "id",
+              "type"
+            ]
+          },
+          "policy": {
+            "request": null,
+            "database": null
+          }
+        },
+        {
+          "action": "create",
+          "fields": {
+            "exclude": [
+              "name"
+            ],
+            "include": [
+              "id"
+            ]
+          },
+          "policy": {
+            "request": null,
+            "database": null
+          }
+        },
+        {
+          "action": "delete",
+          "fields": null,
+          "policy": {
+            "request": null,
+            "database": null
+          }
+        }
       ]
-              }
-            },
-            {
-              "action": "update",
-              "fields": {
-                "include": [],
-                "exclude": [
-                  "*"
-                ]
-              }
-            },
-            "delete"
-          ]
-        },
-        {
-          "role": "authenticated",
-          "actions": [
-            "create",
-            "read",
-            "update",
-            "delete"
-          ]
+    },
+    {
+      "role": "authenticated",
+      "actions": [
+        {
+          "action": "create",
+          "fields": null,
+          "policy": {
+            "request": null,
+            "database": null
+          }
+        },
+        {
+          "action": "read",
+          "fields": null,
+          "policy": {
+            "request": null,
+            "database": null
+          }
+        },
+        {
+          "action": "update",
+          "fields": null,
+          "policy": {
+            "request": null,
+            "database": null
+          }
+        },
+        {
+          "action": "delete",
+          "fields": null,
+          "policy": {
+            "request": null,
+            "database": null
+          }
+        }
+      ]
+    }
+  ],
+  "mappings": null,
+  "relationships": null
+},
+    "Sun": {
+  "source": {
+    "object": "graphqldb.sun",
+    "type": null,
+    "parameters": null,
+    "key-fields": null
+  },
+  "graphql": {
+    "enabled": true,
```

Comment:
I understand in the new config system, all the properties will be written with their default values in the new config files. Im assuming the deserialization logic continues to support starting dab with the old config files. Although all our tests now consume the new config file, we should keep at least some test using the old style of configs - where default properties are NOT written to catch any regressions.

Reply:
Yes, some of the "older" config styles can be found in the tests that have hard-coded config files (as strings) rather than reading them from disk.

It's difficult to have the `dab-config.<DB>` files in the older style because they are the result of the `config-generator` scripts, and we've had a few new steps added so the configs needed to be regenerated, otherwise we wouldn't get consistency between local and CI (since CI generates the config files).